app/src/main/java/com/kickstarter/ui/activities/ThanksActivity.kt

```diff
@@ -55,6 +65,16 @@ class ThanksActivity : AppCompatActivity() {
             orientation = RecyclerView.VERTICAL
         }
 
+        val addressConfiguration = AddressLauncher.Configuration(
+                additionalFields = AddressLauncher.AdditionalFieldsConfiguration(
+                phone = AddressLauncher.AdditionalFieldsConfiguration.FieldConfiguration.REQUIRED
+        ),
+                allowedCountries = setOf("US", "CA", "GB"),
```

Comment:
For the real implementation, perfectly ok on the POC, `config.launchedCountries` will contain the array of allowed countries if the shipping location is worlwide. If the shipping location is restricted we should send only the allowed countries.